server/project-service/src/main/java/edu/illinois/ncsa/incore/service/project/models/VisualizationResource.java

```diff
@@ -109,6 +109,16 @@ public void removeSourceDatasetId(String id) {
         sourceDatasetIds.remove(id);
     }
 
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public String getName() {
+        return name;
+    }
+
```

Comment:
Can you remove the extra line break here?

Reply:
removed